TileStache/Core.py

```diff
@@ -144,12 +144,21 @@
 """
 
 import logging
+from sys import modules
 from wsgiref.headers import Headers
-from StringIO import StringIO
-from urlparse import urljoin
+try:
+    from io import BytesIO
```

Comment:
I can simply wait, since we're not in a hurry on OpenAddresses. I’d also be happy to collaborate on extending tests to cover some of the other cases?